api/api_ethereum_test.go

```diff
@@ -28,6 +32,13 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+var dummyChainConfigForEthereumAPITest = &params.ChainConfig{
+	ChainID:                 new(big.Int).SetUint64(111111),
+	IstanbulCompatibleBlock: new(big.Int).SetUint64(0),
+	LondonCompatibleBlock:   new(big.Int).SetUint64(0),
```

Comment:
Can you add EthTxTypeCompatibleBlock after merging https://github.com/klaytn/klaytn/pull/1175